openapi-communication-service.yaml

```diff
@@ -0,0 +1,714 @@
+        - communication_definition_name: business letter 0001
+          document_type: letter
+          communication_type: SMS_COMMUNICATION
+          communication_data:
+            - key: Customer Name
+              value: Mike Appleseed
+            - key: Account Type
+              value: standard
+            - key: Account Number
+              value: GB 55 BUKB 200415 55555555
+          recipient_phone_number: '+442071838750'
+      description: SMSCommunication model
+      title: SMSCommunication
+    PDFCommunicationBody:
+      x-stoplight:
+        id: mqjzmkd9o40f1
+      allOf:
+        - $ref: '#/components/schemas/CommonCommunicationBody'
+          x-stoplight:
+            id: fq5omqhqvnem9
+        - type: object
+          x-stoplight:
+            id: 4ako3twrl9l4a
+          required:
+            - pdf_delivery_method
+          properties:
+            pdf_delivery_method:
+              x-stoplight:
+                id: pu1zcm36zdhs6
+              type: string
+              x-extensible-enum:
+                - PRINT
+                - NONE
+              description: The way the pdf will be delivered to the recipient
+      examples:
+        - communication_definition_name: business letter 0001
+          document_type: letter
+          communication_type: PDF_COMMUNICATION
+          communication_data:
+            - key: Customer Name
+              value: John Doe
+            - key: Account Type
+              value: standard
+            - key: Account Number
+              value: GB 55 BUKB 200415 55555555
+            - key: Address 1
+              value: 73 Old Edinburgh Road
+            - key: City
+              value: Belfast
+            - key: Post Code
+              value: BT1 3ZQ
+            - key: Country
+              value: United Kingdom
+            - key: SLA
+              value: '1'
+            - key: Environment Code
+              value: P
+            - key: Duplex
+              value: 'Y'
+            - key: Colored
+              value: 'Y'
+            - key: Std Membership Id
+              value: '67432'
+            - key: Envelope Code
+              value: AEID025
+            - key: Post Class
+              value: Whistl
+          pdf_delivery_method: PRINT
+      description: PDFCommunication model
+      title: PDFCommunication
+  securitySchemes:
+    Authorization:
+      type: oauth2
+      flows:
+        authorizationCode:
+          authorizationUrl: 'https://auth.developer.rapidcx.engageone.co/auth2/authorize'
+          tokenUrl: 'https://auth.developer.rapidcx.engage.com/auth2/token'
+          refreshUrl: 'https://auth.developer.rapidcx.engage.com/auth2/token'
+          scopes:
+            read: read-only
+            read-write: read-write
+      description: |-
+        Bearer token required for authentication and authorization.
+
+        Token portal: https://tokens.developer.rapidcx.engageone.co/
+  parameters:
+    RapidCX-Api-Version:
```

Comment:
Shall we provide the example and default version, similarly as here: https://github.com/PreciselyData/rapidcx-public-api/blob/main/openapi-archiver-service.yaml#L237C1-L239C1

Reply:
The value should be 2025-02-13